src/v2/guide/typescript.md

```diff
@@ -0,0 +1,104 @@
+---
+title: TypeScript
+type: guide
+order: 25
+---
+
+If you develop a large application, using a language having static type checking would be helpful. If your code seems to cause unexpected behavior, it provides useful errors before you run it on browsers. You can easily use [TypeScript](https://www.typescriptlang.org/) (one of popular AltJS having static type checking) with Vue since Vue officially provides its declaration files.
```

Comment:
Yeah, it would be clearer and understandable. So how about `For example, if you have typos in your code, ...` ? Because typos are always unexpected thing 😛